feature/matcher/src/main/java/com/simprints/matcher/usecases/FaceMatcherUseCase.kt

```diff
@@ -74,7 +75,7 @@ internal class FaceMatcherUseCase @Inject constructor(
                             loadedCandidates++
                             trySend(MatcherState.CandidateLoaded)
                         }
-                        match(batchCandidates, samples)
+                        bioSdk.createMatcher(samples).use { match(it, batchCandidates) }
```

Comment:
This creates a new matcher for each batch. I suggest moving it outside.

Reply:
What do you mean by "outside"?

Reply:
Sorry, meant to move it outside the batch processing - i.e. above `createRanges()`. This way we'll have only one instance of the matcher that's reused by all batches.